content/tips-on-applying-for-teacher-training.md

```diff
@@ -0,0 +1,179 @@
+7. How long does the training take?
+
+## When to apply
+
+Courses usually open for applications in October for entry the following year
+(usually starting in September, but sometimes January).
+
+Teacher training providers allocate places as people apply through the year,
+and courses stay open until they are full.
+
+If you’re keen to join a popular course it’s wise to apply as soon as you’re
+ready rather than putting it off.
+
+## What to expect: the application form
+
+When you apply you’ll need to give details about:
+
+* your qualifications, including your GCSEs and A levels (or equivalents) and degree
+* your work history or unpaid experience
+* why you want to teach
+* why you’re suited to teach a particular subject or age group
+
+You can also share whether you need any adjustments during the application
+process or on the course - for example, if you’re disabled.
+
+You’ll be encouraged to declare any potential safeguarding issues such as
+criminal convictions. Training providers can advise you whether it’ll affect
+your application.
+
+## Prepare your application
+
+You can [find and apply to most postgraduate teacher training courses on GOV.UK](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+
+If you do not find what you were looking for, there are [other ways to train](/ways-to-train).
+
+Before you start, it’s particularly important to think about your referees and
+personal statement.
+
+Some people do not get on the course they want because their application has
+not convinced the admissions team that they’re right for it.
+
+[Talk to an experienced teacher for help preparing your application.](/tta-service)
+
+## Choose your referees
+
+You need 2 references to apply.
+
+Unlike a standard work reference, your references for teacher training need to
+be named individuals rather than (for example) a human resources team or
+academic department.
+
+Your referees will be asked to write 500 words about your character and
+suitability for teaching.
+
+They’ll have 2 weeks to respond, so it’s worth contacting them before you apply
+so they understand why you’re applying and what they’ll be asked to do.
+
+Choose referees who’ll endorse your:
+
+* teaching passion and potential
+* suitability for working with children
+* academic abilities
+* reliability and professionalism
+
+Ideal referees could include:
+
+* your university tutor or supervisor
+* your current line manager at work
+* your previous employer
+* a teacher at a school where you work or volunteer
+* a supplier or client you’ve worked with (if you’re self-employed)
+
+Referees should not be family members, partners or friends.
+
+Training providers will only accept a character reference as a second reference.
+
+It’s important to have at least one academic or professional reference.
+
+If you’re applying for a salaried course, one of your references must be from
+an employer.
+
+If you’re struggling, get free one-on-one support with choosing your referees.
```

Comment:
here, 'get free one-on-one support...your referees.' should be the TTA service link